docs/usage/configuration-options.md

```diff
@@ -724,6 +724,26 @@ For example, `"baseUrl": "https://api.github.com"` is equivalent to `"hostName":
 
 Renovate does not do a "longest match" algorithm to pick between multiple matching `baseUrl` values in different rules, so put the longer `baseUrl` rule _after_ the shorter one in your `hostRules`.
 
+### concurrentRequestLimit
+
+Usually the default setting is fine, but you can use `concurrentRequestLimit` to limit the number of concurrent
+outstanding requests.
```

Comment:
```suggestion
Usually the default setting is fine, but you can use `concurrentRequestLimit` to limit the number of concurrent outstanding requests.
```

Fix so that we conform to the "one sentence per line" system.

Reply:
Ouch, that's my code reformatting